src/main/java/lessonfour/hometask/dto/UserRegistrationDto.java

```diff
@@ -0,0 +1,17 @@
+package lessonfour.hometask.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.util.Optional;
+
+@ToString
+@Getter
+@AllArgsConstructor
+public class UserRegistrationDto {
+    private String email;
+    private Optional<String> phoneNumber;
```

Comment:
You MUST AVOID using optional as a field. I believe IntelliJ Idea should provide such a warning.